.github/workflows/gh-ci.yaml

```diff
@@ -155,7 +155,7 @@ jobs:
       with:
         micromamba: true
         full-deps: true
-        extra-pip-deps: "docutils sphinx<7 sphinx-sitemap mdanalysis-sphinx-theme>=1.0.1 sphinxcontrib-bibtex pybtex pybtex-docutils"
+        extra-pip-deps: "docutils sphinx<7 sphinx-sitemap mdanalysis-sphinx-theme>=1.3.0 sphinxcontrib-bibtex pybtex pybtex-docutils"
```

Comment:
Note; if these are conda installable now we can just switch this to `extra-conda-deps` to speed up the build.